app/src/main/java/com/github/sdpteam15/polyevents/database/DatabaseConstant.kt

```diff
@@ -55,10 +55,15 @@ object DatabaseConstant {
     /**
      * Area related constants
      */
-    const val ZONE_COLLECTIOn = "zones"
+    const val ZONE_COLLECTION = "zones"
     const val ZONE_DOCUMENT_ID = "zoneId"
     const val ZONE_NAME = "zoneName"
-    const val ZONE_LOCATION = "location"
+    const val ZONE_LOCATION = "zoneLocation"
+    const val ZONE_DESCRIPTION = "zoneDescription"
+    const val LAT_LONG_SEP = "|"
+    const val POINTS_SEP = "!"
+    const val AREAS_SEP = "?"
```

Comment:
Are we sure these characters will never get in the inputs?

Reply:
Yes, probably as the coordinates contains only float, i.e. number containing digit and a point